dag_in_context/src/lib.rs

```diff
@@ -256,7 +256,10 @@ pub enum Schedule {
 #[derive(Clone, Debug)]
 pub struct EggccConfig {
     pub schedule: Schedule,
-    pub stop_after_n_passes: usize,
+    /// Stop after this many passes.
+    /// If stop_after_n_passes is negative,
+    /// run [0 ... schedule.len() + stop_after_n_passes + 1] passes.
```

Comment:
This seems weird... I would expect `-1` to run all but the last pass, but it looks like it runs all of them.
Perhaps you should make it i64::MAX by default and use `[0 .. scheduler.len() + stop_after_n_passes ]` in the neg case?

Reply:
I updated the code to reflect this, although it is a little less elegant now to handle the pass before `i64::MAX`